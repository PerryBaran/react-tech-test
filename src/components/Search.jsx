import React, { useRef } from "react";
import "../styles/search.css";

function Search(props) {
  return (
    <>
      <input type="text" className="search__input"/>
    </>
  );
}

export default Search;
