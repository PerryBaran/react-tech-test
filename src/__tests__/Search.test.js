import { render, screen } from "@testing-library/react";
import Search from "../components/Search";

describe("Search", () => {
  test("snapshot", () => {
    const { asFragment } = render(<Search />);
    expect(asFragment()).toMatchSnapshot();
  });

  test("renders correctly", () => {
    render(<Search />);
    const input = screen.getByRole("textbox");

    expect(input).toBeInTheDocument();
  });
});