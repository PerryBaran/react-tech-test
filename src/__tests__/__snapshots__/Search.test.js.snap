// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search snapshot 1`] = `
<DocumentFragment>
  <input
    class="search__input"
    type="text"
  />
</DocumentFragment>
`;
